src/display/general/overallGraphs.go

```diff
@@ -30,10 +29,52 @@ import (
 	"github.com/pesos/grofer/src/utils"
 )
 
+var myPage *MainPage
 var isCPUSet = false
-
 var run = true
 var helpVisible = false
+var cpuTableVisible = false
+
+func tableChange_3(selectedTable int) {
```

Comment:
the function names here aren't very intuitive, could you either add a doc comment for it or rename if possible?

Reply:
I've added a comment, what should I rename them to ? :thinking: 
tableChange_3 runs when you're switching between 3 tables and tableChange_2 runs when you're switching between only 2 tables.

Reply:
if you've added the comment explaining that then renaming won't be necessary :)